<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>input_image</class>
 <widget class="QWidget" name="input_image">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>193</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="title">
         <string>Use Input Image</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <item row="1" column="0">
          <widget class="QTabWidget" name="tabWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>118</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_7">
            <attribute name="title">
             <string>Input Image</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QPushButton" name="image_thumbnail">
                 <property name="minimumSize">
                  <size>
                   <width>72</width>
                   <height>72</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="acceptDrops">
                  <bool>false</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-radius: 0; background: #fff;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>72</width>
                   <height>72</height>
                  </size>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="2" rowspan="3">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <widget class="QCommandLinkButton" name="import_image_button">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Import image</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="spacing">
                  <number>2</number>
                 </property>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="use_imported_image_button">
                   <property name="minimumSize">
                    <size>
                     <width>26</width>
                     <height>26</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>26</width>
                     <height>26</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>8</pointsize>
                     <bold>false</bold>
                    </font>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="toolTip">
                    <string>Imported input image</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="../../../resources.qrc">
                     <normaloff>:/icons/025-gallery-light.png</normaloff>:/icons/025-gallery-light.png</iconset>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="use_grid_image_button">
                   <property name="minimumSize">
                    <size>
                     <width>26</width>
                     <height>26</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>26</width>
                     <height>26</height>
                    </size>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="toolTip">
                    <string>Active grid input image</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="../../../resources.qrc">
                     <normaloff>:/icons/032-pixels-light.png</normaloff>:/icons/032-pixels-light.png</iconset>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="recycle_grid_image_button">
                   <property name="minimumSize">
                    <size>
                     <width>26</width>
                     <height>26</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>26</width>
                     <height>26</height>
                    </size>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="toolTip">
                    <string>Lock current input image</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="../../../resources.qrc">
                     <normaloff>:/icons/047-recycle-light.png</normaloff>:/icons/047-recycle-light.png</iconset>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="refresh_input_image_button">
                   <property name="minimumSize">
                    <size>
                     <width>26</width>
                     <height>26</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>26</width>
                     <height>26</height>
                    </size>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="toolTip">
                    <string>Refresh current input image</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="../../../resources.qrc">
                     <normaloff>:/icons/050-refresh-light.png</normaloff>:/icons/050-refresh-light.png</iconset>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="clear_image_button">
                   <property name="minimumSize">
                    <size>
                     <width>26</width>
                     <height>26</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>26</width>
                     <height>26</height>
                    </size>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="toolTip">
                    <string>Clear imported input imagea</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="../../../resources.qrc">
                     <normaloff>:/icons/006-trash-light.png</normaloff>:/icons/006-trash-light.png</iconset>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="SliderWidget" name="scale_widget" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliderWidget</class>
   <extends>QWidget</extends>
   <header>airunner/widgets/slider/slider_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>groupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>input_image</receiver>
   <slot>action_toggled_use_input_image(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>7</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>import_image_button</sender>
   <signal>clicked()</signal>
   <receiver>input_image</receiver>
   <slot>action_clicked_button_import_image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refresh_input_image_button</sender>
   <signal>clicked()</signal>
   <receiver>input_image</receiver>
   <slot>action_clicked_button_refresh_grid_image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_image_button</sender>
   <signal>clicked()</signal>
   <receiver>input_image</receiver>
   <slot>action_clicked_button_clear_input_image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recycle_grid_image_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>input_image</receiver>
   <slot>action_toggled_button_refresh_input_image(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recycle_grid_image_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>input_image</receiver>
   <slot>action_toggled_button_lock_grid_image(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_grid_image_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>input_image</receiver>
   <slot>action_toggled_button_use_grid_image(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_imported_image_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>input_image</receiver>
   <slot>action_toggled_button_use_imported_image(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>image_thumbnail</sender>
   <signal>clicked()</signal>
   <receiver>input_image</receiver>
   <slot>action_clicked_button_thumbnail()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>action_toggled_use_input_image(bool)</slot>
  <slot>action_clicked_button_thumbnail()</slot>
  <slot>action_clicked_button_import_image()</slot>
  <slot>action_toggled_button_use_imported_image(bool)</slot>
  <slot>action_toggled_button_use_grid_image(bool)</slot>
  <slot>action_toggled_button_lock_grid_image(bool)</slot>
  <slot>action_clicked_button_refresh_grid_image()</slot>
  <slot>action_clicked_button_clear_input_image()</slot>
  <slot>action_toggled_button_refresh_input_image(bool)</slot>
 </slots>
</ui>
